portfolio/src/main/webapp/nickname-input.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
What is the purpose of this file? Where is it used?

Reply:
After a user clicks the 'Log in' they're redirected to this page, which has no HTML content but prompts the user for their nickname if they haven't set one. I was not able to redirect to the main page and prompt from there because I didn't see a a way to prompt right after checking login status in the LoginServlet file, and waiting until the fetch from the servlet was done resulted in the page continuously refreshing and not loading.